app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -94,6 +96,22 @@ class TopicReviewFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicReviewFragment_loadFragment_selectReviewSkill_configurationChange_conceptCardIsDisplayedCorrectly() {
```

Comment:
Undo this test case. You don't need to write any test case in this file. Instead we have a seperate `ConceptCardFragmentTestActivity` for testing concept card functionality. Please using this file make sure that this activity supports landscape orientation in AndroidManifest file.